### PR TITLE
Change package name for starter cdi project

### PR DESCRIPTION
**Pull request by shadikhani**
Part of vaadin/90min-dx-tasks#51
Should be merged after the origin/dx90-skeleton-starter-flow-cdi-upgrade

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/skeleton-starter-flow-cdi/146)

### CODE REVIEW

File: src/main/java/org/vaadin/example/MainView.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void init() {
 
         // Button click listeners can be defined as lambda expressions
         Button button = new Button("Say hello",
-                e -> Notification.show(greetService.greet(textField.getValue())));
+                e -> Notification.show(GreetService.greet(textField.getValue())));
```
Comment (tmattsso) [inline src/main/java/org/vaadin/example/MainView.java]: again, this should not be static

File: src/main/java/org/vaadin/example/GreetService.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 @VaadinSessionScoped
 public class GreetService {
 
-    public String greet(String name) {
+    public static String greet(String name) {
```
Comment (tmattsso) [inline src/main/java/org/vaadin/example/GreetService.java]: isn't this supposed to be non-static because it is injected using CDI?

Comment (shadikhani) [reply src/main/java/org/vaadin/example/GreetService.java]: I don't know why my changes have been removed. I fixed it.

File: src/main/java/org/vaadin/example/MainView.java (hunk 1 above)
Comment (shadikhani) [reply src/main/java/org/vaadin/example/MainView.java]: Fixed.

**Pull request merged**